Bottom: d246bbc23e3b2d20ab466382344ff4ee8cde588b
Top:    0a60eec4bb6ce18c08a5e832380013687b24ec8f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-21 20:54:31 +0100

Fix rubberbanding not to touch rat-lines


---

diff --git a/src/rubberband.c b/src/rubberband.c
index aec6913..e54d61a 100644
--- a/src/rubberband.c
+++ b/src/rubberband.c
@@ -78,7 +78,7 @@ test_circle_hits_box (PointType *point, Coord radius, BoxType *box)
       box->Y1 - radius <= point->Y && point->Y <= box->Y2 + radius)
     {
       /* We are close enough to the box boundary that we might touch
-       * it, althoguh it is possible that if we are near one of the
+       * it, although it is possible that if we are near one of the
        * box corners, the circle contour doesn't touch the box.
        */
